File: ios/RNMBX/RNMBXImageSourceViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXImageSource, RNMBXImageSourceViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(id, NSString)
RCT_EXPORT_VIEW_PROPERTY(existing, BOOL)
RCT_EXPORT_VIEW_PROPERTY(url, NSString)
RCT_EXPORT_VIEW_PROPERTY(coordinates, NSArray)

@end

File: ios/RNMBX/RNMBXImageSourceViewManager.swift
@objc(RNMBXImageSourceViewManager)
class RNMBXImageSourceViewManager: RCTViewManager {
  @objc
  override static func requiresMainQueueSetup() -> Bool {
    return false
  }

  @objc override func view() -> UIView {
    return RNMBXImageSource()
  }
}
